package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
+  "requires": true,
+  "lockfileVersion": 1,
+  "dependencies": {
+    "ms": {
+      "version": "2.1.2",
+      "resolved": "https://registry.npmjs.org/ms/-/ms-2.1.2.tgz",
+      "integrity": "sha512-sGkPx+VjMtmA6MX27oA4FBFELFCZZ4S4XqeGOXCv68tT+jb3vk/RyaKWP0PTKyWtmLSM0b+adUTEvbs1PEaH2w=="
+    }
+  }
+}
```

Comment:
I think this file was accidentally created 😅

Reply:
Nope, It’s intentional!

Reply:
Hmm, I could'nt find any use case of `ms` package and I'm not sure why need to put this file in root directory.

Reply:
Oops, I missed the path of the file. Sorry about that! should be removed.